example/example.js

```diff
@@ -1,7 +1,5 @@
 // Copyright IBM Corp. 2016. All Rights Reserved.
 // Node module: loopback-connector-informix
-// This file is licensed under the Artistic License 2.0.
```

Comment:
I wonder why these 2 lines are removed.  
For example https://github.com/strongloop/loopback-connector-mysql/blob/master/lib/discovery.js, it has the snippet about "this file is licensed under....".

Reply:
They're removed because in package.json, the `license` is `"SEE LICENSE IN LICENSE"` instead of `Artistic License 2.0`.

Reply:
added them back.